cosmo_tester/resources/blueprints/active-directory/scripts/wait_for_active_directory_port.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+HOST_IP=$public_ip
+PORT=$port
+TIMEOUT_MIN="$(($timeout_minutes * 60))"
```

Comment:
netcat accepts input in seconds. Since I want to convert input from the user, I multiply his input (stated in the bp as timeout_minutes) by 60 to get the result in minutes